server/api/user/user.controller.js

```diff
@@ -0,0 +1,513 @@
+'use strict';
+
+var User = require('./user.model');
+var passport = require('passport');
+var config = require('../../config/environment');
+var jwt = require('jsonwebtoken');
+var crypto = require('crypto');
+var email = require("../../components/email");
+var Commit = require('../commit/commit.model');
+var ClassYear = require('../classyear/classyear.model');
+
+
```

Comment:
just serve up a client folder, for now we only want that, api calls can come after